unit-tests/py/rspy/test.py

```diff
@@ -136,7 +136,7 @@ def print_stack():
     """
     Function for printing the current call stack. Used when an assertion fails
     """
-    log.out( 'Traceback (most recent call last):' )
+    log.e( 'Traceback (most recent call last):' )
```

Comment:
Why should the `print_stack` function fail the test?
The calling function should do that no?

Reply:
I just wanted the stack to show with a red `-E-`, which this does. The stack is only printed on failure.

Reply:
log.e increase the error count
https://github.com/IntelRealSense/librealsense/blob/b874e42685aed1269bc57a2fe5bf14946deb6ede/unit-tests/py/rspy/log.py#L145

So that's why I asked..
You would expect that some error checking will increase it and not a stack print function..

But you saw the output and if you think it's OK I will not object :)

Reply:
I'll check it and make sure

Reply:
In `unit-tests/run-unit-tests.py`, it fails the run if `log.n_errors()` returns non-zero.
But there's no real count per-se. It's just used as a boolean. I'm OK with a stack trace being considered an error there, too.
Also, `run-unit-tests` is not a test, so does not print stack traces -- those are only printed inside each actual test, and `rspy.test` uses a different mechanism (`test.n_failed_tests`) to detect failure.

OK?

Reply:
OK..